docs/modules/ROOT/pages/high-availability-mode.adoc

```diff
@@ -57,4 +57,58 @@ Below are the configuration options for the High Availability Mode feature.
 [source,yaml,subs="attributes+"]
 ----
 include::ROOT:example$/hazelcast-high-availability-mode.yaml[]
-----
\ No newline at end of file
+----
+
+== High Availability Mode and Subset Routing
+
+Subset routing is a client feature that allows a client to connect to a subset of members that is determined by a grouping strategy. To benefit from subset routing, you must set high availability mode, because this parameter is used to determine partition groups. Also, if the subset routing is used on client-side, `exposeExternally.type` must be set to `smart` in the Hazelcast CR.
```

Comment:
```suggestion
`MULTI_MEMBER` routing is a client feature that allows a client to connect to a subset of members that is determined by a grouping strategy. To benefit from `MULTI_MEMBER` routing, you must enable high availability mode, because this parameter is used to determine partition groups. Also, if the `MULTI_MEMBER` routing mode is used on the client-side, `exposeExternally.type` must be set to `smart` in the Hazelcast CR.
```